### ISSUE TITLE
loss and accuracy was not normal when trained with 30 epochs

### ISSUE THREAD

**Issue by yangxiuwu** [Bartzi/stn-ocr#3]
![plot](https://user-images.githubusercontent.com/17425009/29763313-2f465e34-8c05-11e7-8f66-cb82da9767f4.png)
 I used generated/centered dataset,  and I confused about why there was no validation  information in the log file.

**Comment by Bartzi** [comment]
hmm, in order to help you I might need the log output that was generated during your training, because the plot directly depends on the log output, as we have seen in your last [issue](https://github.com/Bartzi/stn-ocr/issues/2). :wink:

**Comment by yangxiuwu** [comment]
[log.gz](https://github.com/Bartzi/stn-ocr/files/1257276/log.gz)
thanks, here is the log.

**Comment by Bartzi** [comment]
Well that is interesting, if I'm using the `plot_log.py` script in the folder `mxnet/utils` without any changes in your log file I get the following output:

![plot](https://user-images.githubusercontent.com/2399349/29811427-ae5a4854-8ca3-11e7-8ec6-8fe4b87ed56b.png)

I did the following:
`python plot_log.py <path to log file> -d plot.png`.
So the validation information is clearly in the log file. The first validation occurence is in line 406 of your file.

Did you do the same as I did?

**Comment by yangxiuwu** [comment]
I'm very sorry for my careless, I'm not check the log carefully. when I debug the script with newest version of mxnet there was an error with plot_log.py, so I comment the line of `axe.plot(x_test, [test_iterations[iteration][metric] for iteration in x_test], 'g.-', label='test')`. now it can plot out the validation curve. So the only question is why the accuracy was decline after some epochs.

**Comment by Bartzi** [comment]
Alright, well I'm not 100% sure why this happens, but my guess is that the learning rate was too high, s.th. like this usually happens because of that... so it could be that the network jumped out of the local minimum/saddle point it was currently in and was not able to find a way back...

**Comment by yangxiuwu** [comment]
Your guess is right. I adjust learning rate from 1e-5 to 1e-6, then the accuracy trend line become normal. Thanks for your help.
